octobot_trading/personal_data/portfolios/history/historical_portfolio_value_manager.py

```diff
@@ -77,11 +84,21 @@ async def on_new_value(self, timestamp, value_by_currency, force_update=False, s
         timestamp
         :return: True if something changed
         """
+        self.last_update_time = self.portfolio_manager.exchange_manager.exchange.get_exchange_current_time()
+        previous_ending_portfolio = self.ending_portfolio
+        self.ending_portfolio = portfolio_util.portfolio_to_float(
+            self.portfolio_manager.portfolio.portfolio
+        )
```

Comment:
Do we really want to do this every time the portfolio is updated?

Reply:
I changed it to be updated only on portfolio update